src/views/forms/TradeForm/AdvancedTradeOptions.tsx

```diff
@@ -129,36 +129,37 @@ export const AdvancedTradeOptions = () => {
                 ))}
               </Styled.SelectMenu>
             )}
-            {needsPostOnly && (
-              <Checkbox
-                checked={postOnly || false}
+            {(needsReduceOnly || reduceOnlyTooltip) && <Checkbox
+                checked={(reduceOnly && !reduceOnlyTooltip) || false}
+                disabled={!!reduceOnlyTooltip}
                 onCheckedChange={(checked) =>
                   abacusStateManager.setTradeValue({
                     value: checked,
-                    field: TradeInputField.postOnly,
+                    field: TradeInputField.reduceOnly,
                   })
                 }
-                id="post-only"
+                id="reduce-only"
                 label={
-                  <WithTooltip tooltip="post-only" side="right">
-                    {stringGetter({ key: STRING_KEYS.POST_ONLY })}
+                  <WithTooltip tooltip={needsReduceOnly ? "reduce-only" : reduceOnlyTooltip?.titleStringKey.includes("REDUCE_ONLY_EXECUTION_IOC_FOK") ? "reduce-only-execution-ioc-fok" : "reduce-only-timeinforce-ioc-fok"} side="right">
```

Comment:
I don't love this but wasn't sure how to otherwise differentiate :( Unfortunately can't directly render what's returned by `reduceOnlyTooltip` - (we get something like `GENERAL.TRADE.REDUCE_ONLY_TIMEINFORCE_IOC_FOK.BODY`)